composer.json

```diff
@@ -33,8 +36,8 @@
         }
     },
     "require": {
-        "php": "^7.2.5|^8.0",
-        "guzzlehttp/guzzle": "^7.0",
+        "php": "~7.0 || ^8.0",
```

Comment:
are we going down in version here? before it was `^7.2.5` and now it's 7.0?

Reply:
I can't recall why/how this changed but I'll set it back